meshtastic/interfaces/ble/runner.py

```
@@ -569,8 +569,8 @@ def _handle_loop_exception(
         try:
             loop.default_exception_handler(context)
         except (
-            Exception  # noqa: BLE001 - loop default handler failures are non-fatal
-        ) as e:
+            Exception
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)